Configure

```diff
@@ -581,7 +581,7 @@ my @disable_cascades = (
 
     "module"            => [ "fips", "dso" ],
 
-    "engine"            => [ grep /eng$/, @disablables ],
+    "engine"            => [ grep /(eng$|dynamic-engine)/, @disablables ],
```

Comment:
Uhm, that looks overly incomprehensible.  Why not this?

``` perl
    "engine"            => [ "dynamic-engine", grep(/eng$/, @disablables) ],
```

Reply:
Yeah. That works too. I've changed it to look like that.